Components/Button/stories.jsx

```diff
@@ -2,17 +2,20 @@ import React from "react";
 import { storiesOf, setAddon } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 import chaptersAddon from 'react-storybook-addon-chapters';
+import { doc } from 'storybook-readme';
+import README from './README.md';
 import Button from "./index";
 
 setAddon(chaptersAddon);
-
-storiesOf("Button", module).addWithChapters('Display',  {
+storiesOf('Button', module).add('Documentation', doc(README));
```

Comment:
The word 'documentation' seems a bit tautological, maybe 'overview' would be better?

Reply:
Good point!

![](https://imgs.xkcd.com/comics/honor_societies.png)

I'll change.

Reply:
lol, that's a funny one 😄